unit-tests/func/options/test-color-options.py

```diff
@@ -0,0 +1,95 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Rename to : `test-rgb-options-metadata-consistency` or similar

Reply:
Done